db/migrate/20210324174033_create_topics.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTopics < ActiveRecord::Migration[6.1]
+  def up
+    create_table :topics do |t|
+      t.string 'description', limit: 50, null: false
+
+      t.timestamps
+    end
+  end
+
+  def down
+    drop_table :topics
+  end
```

Comment:
you don't need to do an up and a down method, just using change method is enough 😄